src/templates/Footer/index.js

```diff
@@ -80,10 +99,18 @@ Footer.propTypes = {
    * Function to trigger chat.
    */
   onClickChat: PropTypes.func,
+  /**
+   * Array of Links `[ { text: 'link text', href: 'http://link.com' } ]`
+   */
+  links: PropTypes.array,
```

Comment:
could do `PropTypes.arrayOf(PropTypes.shape({ text: PropTypes.string, href: PropTypes.string.isRequired }))`. it's verbose, but could be helpful for ensuring an `href` is passed for every link